infra/base-images/base-runner/coverage

```diff
@@ -99,7 +99,12 @@ function run_fuzz_target {
     return 0
   fi
 
-  llvm-profdata merge -j=1 -sparse $profraw_file_mask -o $profdata_file
+  llvm-profdata merge -j=1 -sparse $profraw_file_mask -o $profdata_file || (
+    # on error, try to first run llvm-cov-rel to fix profraw file
```

Comment:
This is pretty hacky to me. Why can't you just do the latter when using rust or swift?
Or would doing the latter for every language work.

Reply:
Please begin with capital and end with period.

Reply:
Yes, or we can do it in every case, as it leaves clang-14 profraw files identical